packages/account-v2/src/containers/TradingAssessmentForm/TradingAssessmentForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Form, Formik } from 'formik';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { getTradingAssessmentQuestions } from '../../constants/tradingAssessmentQuestions';
+import { RiskToleranceWarningModal } from './RiskToleranceWarningModal';
+import { TestWarningModal } from './TestWarningModal';
+import { TradingAssessmentContainer } from './TradingAssessmentContainer';
+
+export const TradingAssessmentForm = () => {
```

Comment:
is this form supposed to be used only in account settings or in account creation flow as well?

Reply:
Yes this form will be pass to real account account creation

Reply:
no, I meant the trading assessment question in financial assessment page. (if we will show it :) )